cmd/apps/harbor.go

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) arkade author(s) 2022. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package apps
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/alexellis/arkade/pkg"
+	"github.com/alexellis/arkade/pkg/apps"
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/types"
+)
+
+func MakeInstallHarbor() *cobra.Command {
+	var harbor = &cobra.Command{
+		Use:   "harbor",
+		Short: "Install harbor",
+		Long:  `Install harbor`,
+		Example: `  arkade install harbor
+  # with ingress and custom domain
+  arkade install harbor --ingress=true --domain=example.com`,
+		SilenceUsage: true,
+	}
+
+	harbor.Flags().StringP("namespace", "n", "harbor", "The namespace to install the chart")
+	harbor.Flags().Bool("update-repo", true, "Update the helm repo")
+	harbor.Flags().Bool("ingress", false, "Enable ingress")
+	harbor.Flags().Bool("trivy", false, "Enable trivy")
+	harbor.Flags().Bool("notary", false, "Enable notary")
```

Comment:
Notary was removed as of release 2.8 we have replaced it with cosign!

Reply:
Thanks for the info @OrlinVasilev 

Anything else we need to know?

@Jasstkn want to swap it out?

Reply:
that looks ok for now! Cosign comes by default